xml/ha_maintenance.xml

```diff
@@ -0,0 +1,354 @@
+   <step>
+    <para>
+     Open the <guimenu>Meta Attributes</guimenu> category.
+    </para>
+   </step>
+   <step>
+    <para>
+     From the empty drop-down list, select the <guimenu>maintenance</guimenu>
+     attribute and click the plus icon to add it.
+    </para>
+   </step>
+   <step>
+    <para>
+     Activate the check box next to <literal>maintenance</literal> to set the
+     maintenance attribute to <literal>yes</literal>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Confirm your changes.
+    </para>
+   </step>
+   <step>
+    <para>
+     After you have finished the maintenance task for that resource, deactivate
+     the check box next to the <literal>maintenance</literal> attribute for
+     that resource.
+    </para>
+    <para>
+     From this point on, the resource will be managed by the &hasi; software
+     again.
+    </para>
+   </step>
+  </procedure>
+
+  <procedure xml:id="pro.ha.config.hawk.rsc.is-managed">
+   <title>Putting a Resource Into Unmanaged Mode</title>
+   <step>
+    <para>
+     Start a Web browser and log in to the cluster as described in
+     <xref
+      linkend="sec.conf.hawk2.login"/>.
+    </para>
+   </step>
+   <step>
+    <para>
+     From the left navigation bar, select <guimenu>Status</guimenu> and go to
+     the <guimenu>Resources</guimenu> list.
+    </para>
+   </step>
+   <step>
+    <para>
+     In the <guimenu>Operations</guimenu> column, click the arrow down icon
+     next to the resource you want to modify and select
+     <guimenu>Edit</guimenu>.
+    </para>
+    <para>
+     The resource configuration screen opens.
+    </para>
+   </step>
+   <step>
+    <para>
+     Below <guimenu>Meta Attributes</guimenu>, select the
+     <guimenu>is-managed</guimenu> entry from the empty drop-down box.
+    </para>
+   </step>
+   <step>
+    <para>
+     Set its value to <literal>No</literal> and click <guimenu>Apply</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     After you have finished your maintenance task, set
+     <guimenu>is-managed</guimenu> to <literal>Yes</literal> (which is the
+     default value) and apply your changes.
+    </para>
+    <para>
+     From this point on, the resource will be managed by the &hasi; software
+     again.
+    </para>
+   </step>
+  </procedure>
+ </sect1>
+ <sect1 xml:id="sec.ha.manual_config.cli.maint.mode">
+  <title>Using Maintenance Modes with &crmsh;</title>
+
+  <variablelist>
+   <varlistentry>
+    <term>Applying Maintenance Mode to your Cluster</term>
+    <listitem>
+     <para>
+      In case you want to put the whole cluster in maintenance mode, use the
+      following command:
+     </para>
+<screen>&prompt.root;<command>crm</command> configure property maintenance-mode=true</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Applying Maintenance Mode to Nodes</term>
+    <listitem>
+     <para>
+      For example, to put the node <literal>&node1;</literal> into maintenance
```

Comment:
From a concistency point of view, you could move the "For example" at the end of the sentence (as you've started all other items with "To put...").

Reply:
Done.